source/_integrations/sql.markdown

```diff
@@ -25,7 +25,7 @@ This can be used to present statistics about Home Assistant sensors if used with
 
 See [supported engines](/integrations/recorder/#custom-database-engines) for which you can connect with this integration.
 
-The SQL integration will connect to default recorder if Database URL is not specified.
+The SQL integration will connect to SQLite if Database URL is not specified. If you use a different database recorder (eg MariaDB), you will have to specify the Database URL.
```

Comment:
```suggestion
The SQL integration will connect to the default SQLite if "Database URL" has not been specified. If you use a different database recorder (eg MariaDB or others), you will have to specify the "Database URL" manually during integration setup.
```

Reply:
Sorry I missed the boxes, hope I fixed it now. And your wording is better than mine. Do I need do do anything else?

Reply:
You can either commit my suggestion or you update the markdown in your branch on your fork so it comes through.
Also in the PR description you should mark the issue as "fixes" to it will close on merging this PR